### DISCUSSION TITLE
Up-to-date deployment guide for Digital Ocean App

### DISCUSSION THREAD

**Discussion by TadasV1** [sveltejs/kit#10434] [General]
Trying to deploy on digital ocean. 

Keep getting all kind errors. Currently stuck:
Deploy Error: Container Terminated
ERROR: failed to launch: determine start command: process type web was not

Have no idea what causes. Some say its port issue. 

Anyone successfully deployed on DO App?

**Comment by ramonmalcolm10** [comment]
I did a successfully deployment a few months back. Are you using adapter node?

**Reply by TadasV1** [reply]
Started after adapter-auto through error message. I assumed it will recognized env and will load node-adapter. 
Also missed 'node build' run command on digital ocean... now all deployed.
